build/build-sdk-images/node/Dockerfile

```diff
@@ -14,14 +14,13 @@
 ARG BASE_IMAGE=agones-build-sdk-base:latest
 FROM $BASE_IMAGE
 
-## --allow-releaseinfo-change is a workaround for the new debian release
-## Can be removed when upgrading to bullseye
-RUN apt-get --allow-releaseinfo-change update && curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
-    apt-get install -y nodejs
+COPY ./examples/nodejs-simple .
```

Comment:
I'm not sure why we do this here, should not be required to build the SDK or are we running the simple example as a kind of test?